docs/wiki/tuning/4-2-Tuning-Notes.md

```diff
@@ -6,37 +6,37 @@ sidebar_position: 11
 
 **4.2 should fly really well on defaults, with no modifications, with most quads.** If you know your quad requires special or unusual PIDs, please re-tune, because with the new 4.2 features, you may get you an even better result with different PIDs. For instance, if you wanted to be able to add more D but were limited by noise, it may be possible now.
 
-**Some low authority quads (under-powered whoops, ducted HD whoops, 7" and larger quads, endurance quads, etc) may require [specific settings to minimise bounce-back](4.2-Tuning-Notes#settings-to-minimise-bounceback).**
+**Some low authority quads (under-powered whoops, ducted HD whoops, 7" and larger quads, endurance quads, etc) may require [specific settings to minimise bounce-back](#settings-to-minimise-bounceback).**
 
 If you're new to 4.2, please read the 4.0 and 4.1 tuning notes.
 
 **Configurator version 10.7 or higher is mandatory. Get it from [Configurator Releases.](https://github.com/betaflight/betaflight-configurator/releases). Earlier versions are not going to work properly.**
 
 **NOTE 1**: DO NOT paste a diff or dump from any prior build into the 4.2 CLI. See if you can use the sliders to approximate your old PIDs and filter settings, to make it easier, in future, to use sliders for tuning.
 
-**NOTE 2**: If you previously set `dyn_notch_range` to `LOW`, change `dyn_notch_max_hz` to 350, and for `HIGH` to 700. Default of 500 for most quads is fine otherwise. Read more about setting the range [here](4.2-Tuning-Notes#improved-dynamic-notch-filter).
+**NOTE 2**: If you previously set `dyn_notch_range` to `LOW`, change `dyn_notch_max_hz` to 350, and for `HIGH` to 700. Default of 500 for most quads is fine otherwise. Read more about setting the range [here](#improved-dynamic-notch-filter).
 
-**NOTE 3**:If your previous PID `I` values were changed from default, the anti-gravity gain values in 4.2 may need changing a little to get the same outcome. We recommend flying the default antigravity value and re-tuning it to the optimal value. Details [here](4.2-Tuning-Notes#antigravity-gain-independent-of-I-values).
+**NOTE 3**:If your previous PID `I` values were changed from default, the anti-gravity gain values in 4.2 may need changing a little to get the same outcome. We recommend flying the default antigravity value and re-tuning it to the optimal value. Details [here](#antigravity-gain-independent-of-I-values).
 
 **NOTE 4**: If using the Accelerometer, arming will fail unless it has been calibrated.
 
-**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](/docs/wiki/guides/current/DSHOT-RPM-Filtering)
+**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](docs/wiki/guides/current//DSHOT-RPM-Filtering)
```

Comment:
```suggestion
**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](docs/wiki/guides/current/DSHOT-RPM-Filtering)
```

Reply:
Searched and fixed all similar ones, thanks for the heads up

Reply:
fixed